docs/supported_docker_environment/logging_config.md

```diff
@@ -18,5 +18,14 @@ should be included in your classpath to show a reasonable level of log output:
 
     <logger name="org.testcontainers" level="INFO"/>
     <logger name="com.github.dockerjava" level="WARN"/>
+    <logger name="org.apache.hc.client5.http.wire" level="OFF"/>
```

Comment:
```suggestion
    <logger name="com.github.dockerjava.zerodep.shaded.org.apache.hc.client5.http.wire" level="OFF"/>
```

Since nowadays the client is shared into `docker-java`, this is the correct logger name.

Reply:
Thanks for the fix @kiview :)